.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
 ignore = E203, E266, E501, W503
 max-line-length = 88
-max-complexity = 20
+max-complexity = 18
```

Comment:
Did you mean to commit this? Is is better t reduce complexity?

Reply:
That's the number of nested statements, lower is better for code readability.